src/plugins/ConditionalRule.js

```diff
@@ -18,6 +19,7 @@ export default class ConditionalRule {
 
   constructor(selector: string, styles: Object, options: RuleOptions) {
     this.selector = selector
+    this.selectorId = selector.replace(' ', '')
```

Comment:
here is the same trick https://github.com/cssinjs/jss/pull/512/files#diff-aeadf25904d143f46b87eef264b77f87R22